lib/brine/rest_steps.rb

```diff
@@ -71,21 +67,11 @@ def kv_table(table)
 #
 # Response body assertions
 #
+# TODO: Write specifications around this
```

Comment:
This comment is vague...this case should already be supported so this step could be deprecated if specs are created to prove that.